go/vt/sqlparser/comments.go

```diff
@@ -32,6 +32,8 @@ const (
 	DirectiveQueryTimeout = "QUERY_TIMEOUT_MS"
 	// DirectiveScatterErrorsAsWarnings enables partial success scatter select queries
 	DirectiveScatterErrorsAsWarnings = "SCATTER_ERRORS_AS_WARNINGS"
+	// DirectiveMaxPayloadSize sets a maximum payload size in bytes. Only supported for INSERTS.
```

Comment:
What would you think about also supporting it for updates?

Reply:
Resolving this comment as the new implementation covers inserts/updates/selects/deletes!